HelloWorld/Main.java

```diff
@@ -4,6 +4,14 @@
 {
     public static void main(String[] args)
     {
+
         System.out.println("Hello world");
+        System.out.println("System.out.println command use to display something on the console");
```

Comment:
This source code may be into other Java program, for example, a java file called Calculate.java